src-php/Routes/web.php

```diff
@@ -5,8 +5,18 @@
  */
 Route::middleware('web')->prefix('events')->name('events.')->group(function () {
     Route::get('/', 'Dewsign\NovaEvents\Http\Controllers\EventController@index')->name('index');
-    Route::get('/archive', 'Dewsign\NovaEvents\Http\Controllers\EventController@archive')->name('archive');
-    Route::get('/by-date', 'Dewsign\NovaEvents\Http\Controllers\EventController@byDate')->name('by-date');
+    if (config('nova-events.enableArchive')) {
+        Route::get('/archive', 'Dewsign\NovaEvents\Http\Controllers\EventController@archive')->name('archive');
+    } else {
+        Route::redirect('/archive', '/events');
```

Comment:
if this is even a good idea, should probably make it a 301 right?

Or 302 as it maybe enabled again later.